infra/modules/azure_cdn/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "id" {
```

Comment:
Always add `name` and `resource_group_name` as they are the most used module outputs